packages/@okta/vuepress-theme-prose/global-components/StackSelector.vue

```diff
@@ -50,24 +51,27 @@
         if (value && value.link) {
           this.hasFocus = true;
           this.$router.push(value.link);
+          this.stackSelectorData.from = this.selectedOption.link;
```

Comment:
Maybe leave a comment here similar to `SidebarItem` to explain why this data is added?